docs/src/components/home/DiamondSponsors.tsx

```diff
@@ -18,8 +18,8 @@ const DIAMONDs = [
   {
     src: '/static/sponsors/doit-square.svg',
     name: 'Doit International',
-    description: 'Management platform for Google Cloud and AWS',
-    href: 'https://www.doit.com/flexsave/?utm_source=MUI&utm_medium=referral&utm_content=homepage',
+    description: 'Delivers technology and cloud expertise to buy, optimize and manage public cloud',
```

Comment:
```suggestion
    description: 'Delivers technology and cloud expertise to buy, optimize and manage public cloud.',
```

Just missing a period there 👍